lib/bots/notifybot.ts

```diff
@@ -0,0 +1,871 @@
+/*
+Copyright 2017 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import * as Promise from 'bluebird';
+import { Front, TopicConversations } from 'front-sdk';
+import * as GithubApi from 'github';
+import * as yaml from 'js-yaml';
+import * as _ from 'lodash';
+import * as GithubApiTypes from '../apis/githubapi-types';
```

Comment:
I guess we could. I'll raise an issue for it though globally for later, rather than go through them all for this PR.